### PR TITLE
Code Quality Fixes

### PR DESCRIPTION
**Pull request by lakshyacis**
filesystem flake8 fixes
filesystem pylint fixes
sqlite flake8 fixes
sqlite pylint fixes
storage flake8 fixes
storage pylint fixes

### CODE REVIEW

**Comment by g1itch** [comment]
It seems you don't understand the warnings and don't read existing PR's. Look: https://github.com/g1itch/PyBitmessage/commit/34a4e19226ce0bb028aa2a881317a4e7d23261d8.

My branch is not ideal but I hope illustrative enough.

**Comment by lakshyacis** [comment]
> It seems you don't understand the warnings and don't read existing PR's. Look: [g1itch@34a4e19](https://github.com/g1itch/PyBitmessage/commit/34a4e19226ce0bb028aa2a881317a4e7d23261d8).
> 
> My branch is not ideal but I hope illustrative enough.

Okay, will try to improvise the code and will update soon. Thanks

**Comment by g1itch** [comment]
Less `# pylint disable=` - much better. It seems the value of variable you named `hashval` is the same as `hashId` in other places.

**Comment by lakshyacis** [comment]
> Less `# pylint disable=` - much better. Also it seems the value of variable you named `hashval` is the same as `hashId` in other places.

So, if i use hash_, then it will be okay?

**Comment by g1itch** [comment]
> So, if i use hash_, then it will be okay?

I think that vars should have the same names. The exact name is not so important but `invhash` seems more meaningful. Maybe ask @PeterSurda?

File: src/storage/storage.py [hunk 1]
```diff
@@ -31,18 +37,24 @@ def __len__(self):
         raise NotImplementedError
 
     def by_type_and_tag(self, objectType, tag):
+        """"""
```
Comment (PeterSurda) [inline src/storage/storage.py]: `"""Return inventory vectors filtered by object type and tag"""`

File: src/storage/storage.py [hunk 2]
```diff
@@ -31,18 +37,24 @@ def __len__(self):
...
         raise NotImplementedError
 
     def unexpired_hashes_by_stream(self, stream):
+        """"""
```
Comment (PeterSurda) [inline src/storage/storage.py]: `"""Return unexpired inventory vectors filtered by stream"""`

File: src/storage/storage.py [hunk 3]
```diff
@@ -31,18 +37,24 @@ def __len__(self):
...
         raise NotImplementedError
 
     def flush(self):
+        """"""
```
Comment (PeterSurda) [inline src/storage/storage.py]: `"""Flush cache"""`

File: src/storage/storage.py [hunk 4]
```diff
@@ -31,18 +37,24 @@ def __len__(self):
...
         raise NotImplementedError
 
     def clean(self):
+        """"""
```
Comment (PeterSurda) [inline src/storage/storage.py]: `"""Free memory / perform garbage collection"""`

File: src/storage/storage.py [hunk 5]
```diff
@@ -1,27 +1,33 @@
...
+======================
+"""
 import collections
 
 InventoryItem = collections.namedtuple('InventoryItem', 'type stream payload expires tag')
 
-class Storage(object):
+
+class Storage(object):    # pylint: disable=too-few-public-methods
+    """"""
```
Comment (PeterSurda) [inline src/storage/storage.py]: `"""Base class for storing inventory (extendable for other items to store)"""

File: src/storage/filesystem.py [hunk 6]
```diff
@@ -1,94 +1,128 @@
...
         self.baseDir = path.join(lookupAppdataFolder(), FilesystemInventory.topDir)
         for createDir in [self.baseDir, path.join(self.baseDir, "objects")]:
             if path.exists(createDir):
                 if not path.isdir(createDir):
                     raise IOError("%s exists but it's not a directory" % (createDir))
             else:
                 makedirs(createDir)
-        self.lock = RLock() # Guarantees that two receiveDataThreads don't receive and process the same message concurrently (probably sent by a malicious individual)
+        self.lock = RLock()
+        # Guarantees that two receiveDataThreads don't receive and process the same message
```
Comment (PeterSurda) [inline src/storage/filesystem.py]: This refers to `self.lock` and according to PEP8 block comments should be **above** the code they refer to.

**Comment by PeterSurda** [comment]
I don't particularly care about the name, as long as it's obvious what it's doing, All three options mentioned, `hashval`, `hashId` and `hash_` are ok.

File: src/storage/filesystem.py [hunk 7]
```diff
@@ -103,44 +137,66 @@ def _load(self):
...
+                ),
+                "r",
+            ) as f:
                 objectType, streamNumber, expiresTime, tag, undef = string.split(f.read(), ",", 4)
                 return [int(objectType), int(streamNumber), int(expiresTime), unhexlify(tag)]
         except IOError:
             raise KeyError
 
     def by_type_and_tag(self, objectType, tag):
+        """Get object type and tag"""
```
Comment (PeterSurda) [inline src/storage/filesystem.py]: `"""Get a list of inventory vectors filtered by object type and tag"""`

File: src/storage/sqlite.py [hunk 8]
```diff
@@ -1,45 +1,59 @@
...
+class SqliteInventory(InventoryStorage):    # pylint: disable=too-many-ancestors
+    """Inventory using SQLite"""
     def __init__(self):
-        super(self.__class__, self).__init__()
-        self._inventory = {} #of objects (like msg payloads and pubkey payloads) Does not include protocol headers (the first 24 bytes of each packet).
-        self._objects = {} # cache for existing objects, used for quick lookups if we have an object. This is used for example whenever we receive an inv message from a peer to check to see what items are new to us. We don't delete things out of it; instead, the singleCleaner thread clears and refills it.
-        self.lock = RLock() # Guarantees that two receiveDataThreads don't receive and process the same message concurrently (probably sent by a malicious individual)
+        super(SqliteInventory, self).__init__()
+        self._inventory = {}
+        # of objects (like msg payloads and pubkey payloads)
```
Comment (PeterSurda) [inline src/storage/sqlite.py]: This comment should be moved up, it refers to `self._inventory`.

File: src/storage/sqlite.py [hunk 9]
```diff
@@ -1,45 +1,59 @@
...
+        # Does not include protocol headers (the first 24 bytes of each packet).
+        self._objects = {}
+        # cache for existing objects, used for quick lookups if we have an object.
```
Comment (PeterSurda) [inline src/storage/sqlite.py]: Also move up.

File: src/storage/sqlite.py [hunk 10]
```diff
@@ -1,45 +1,59 @@
...
+        # This is used for example whenever we receive an inv message from a peer
+        # to check to see what items are new to us.
+        # We don't delete things out of it; instead, the singleCleaner thread clears and refills it.
+        self.lock = RLock()
+        # Guarantees that two receiveDataThreads don't receive and process the same message concurrently
```
Comment (PeterSurda) [inline src/storage/sqlite.py]: Also move up

File: src/storage/storage.py (hunk 1 above)
Comment (lakshyacis) [reply src/storage/storage.py]: Okay will update, thanks!

File: src/storage/storage.py (hunk 2 above)
Comment (lakshyacis) [reply src/storage/storage.py]: Okay, thanks!

File: src/storage/storage.py (hunk 4 above)
Comment (lakshyacis) [reply src/storage/storage.py]: Okay.

File: src/storage/filesystem.py (hunk 6 above)
Comment (lakshyacis) [reply src/storage/filesystem.py]: Okay, will amend the changes. Thank you.

File: src/storage/sqlite.py (hunk 10 above)
Comment (lakshyacis) [reply src/storage/sqlite.py]: okay

**Comment by lakshyacis** [comment]
> Please make the changes as requested.

Done!

**Pull request merged**